setup.py

```diff
@@ -60,7 +60,7 @@ def findsome(subdir, extensions):
     packages=[pkg for pkg in find_packages('.') if pkg.startswith('datalad')],
     # datalad command suite specs from here
     install_requires=[
-        'datalad[full]>=0.12.0.rc3',
+        'datalad[full]>=0.10.0.rc1',
```

Comment:
if you decide to go this route, please adjust this along with it to be `>= 0.11` so nobody pulls in rc's with it.

Reply:
Agree.

Reply:
```suggestion
        'datalad[full]>=0.11',
```